ServiceDocumentSampleComponents/serviceDocumentSampleComponents/force-app/main/default/lwc/demoWorkStepPresentation/__tests__/demoWorkStepPresentation.test.js

```diff
@@ -0,0 +1,25 @@
+import { createElement } from "lwc";
+import DemoWorkStepPresentation from "c/demoWorkStepPresentation";
+
+describe("c-demo-work-step-presentation", () => {
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it("TODO: test case generated by CLI command, please fill in test logic", () => {
+    // Arrange
+    const element = createElement("c-demo-work-step-presentation", {
+      is: DemoWorkStepPresentation
+    });
+
+    // Act
+    document.body.appendChild(element);
+
+    // Assert
+    // const div = element.shadowRoot.querySelector('div');
+    expect(1).toBe(1);
+  });
+});
```

Comment:
For these boilerplate test files - I'd rather we remove them than have them hanging around uselessly